images/sponsors/code-ninjas-logo.svg

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 21.0.2, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```

Comment:
Try and clean up some of this excess data—there are a lot of elements that don't need to be here.